### DISCUSSION TITLE
When React-Query V4 will be released?

### DISCUSSION THREAD

**Discussion by nguyenyou** [TanStack/query#3750] [General] [answered]
I'm really excited about the `persistQueryClient` feature and waiting for React-Query V4 stable release to apply it in our production website. Can we expect a release date of React-Query V4? Btw, thanks so much for all the hard work 💯

**Comment by TkDodo** [comment, marked as answer]
We'e in the middle of figuring out the re-brand to Tanstack Query. Functionality wise, we are done :)

**Reply by nguyenyou** [reply]
Thank you 👍

**Comment by HXavS** [comment]
@TkDodo, Yeah the new site looks great. https://tanstack.com/query/v4. 

But was a little confused with the Get Started not talking about beta, if you follow them you will install v3 when expecting v4.
